plugins/@grouparoo/app-templates/src/source/table/records.ts

```diff
@@ -82,7 +84,10 @@ export const getRecords: GetRecordsMethod = ({ getChangedRows }) => {
       const currentValue = highWaterMark[highWaterMarkAndSortColumnASC];
       const newValue = lastRow[highWaterMarkKey].toString();
 
-      if (currentValue && newValue === currentValue) {
+      if (!schedule.incremental) {
+        nextSourceOffset = sourceOffset + limit;
+        nextHighWaterMark[highWaterMarkAndSortColumnASC] = newValue;
```

Comment:
is this right? This would change the `where` which out make the limit/offset different.
I would think it would only be `nextSourceOffset = sourceOffset + limit;`

Reply:
We decided to keep storing the highWaterMarks, even though we aren't using them.  This way, you could choose to turn on  incremental later and use the most recent highWaterMark from the last run